templates/adherent/_list_my_sections.html.twig

```diff
@@ -102,3 +102,10 @@
         </a>
     </li>
 {% endif %}
+{% if app.user.isLre %}
```

Comment:
Should we check for `ROLE_LRE` instead?

Reply:
I did the same as the other checks, anyway, in this case, this is the same, as ROLE_LRE is present when isLre returns true :)